### PR TITLE
Slow Falling Effect

### PR DESCRIPTION
**Pull request by uselesswaifu**
The color thing was kinda confusing. Feel free to change it

## Introduction
 The Slow falling effect is missing. So I added it

### CODE REVIEW

File: src/pocketmine/entity/Effect.php [hunk 1]
```diff
@@ -91,6 +92,7 @@ public static function init() : void{
 		self::registerEffect(new Effect(Effect::LEVITATION, "%potion.levitation", new Color(0xce, 0xff, 0xff)));
 		self::registerEffect(new Effect(Effect::FATAL_POISON, "%potion.poison", new Color(0x4e, 0x93, 0x31), true));
 		self::registerEffect(new Effect(Effect::CONDUIT_POWER, "%potion.conduitPower", new Color(0x1d, 0xc2, 0xd1)));
+		self::registerEffect(new Effect(Effect::SLOW_FALLING, "%potion.slowFalling", new Color(206, 255, 255)));
```
[the remark this answers is not in the document]
Comment (uselesswaifu) [reply src/pocketmine/entity/Effect.php]: Thanks! I've changed it

**Comment by dktapps** [comment]
This is not a complete implementation.

**Comment by uselesswaifu** [comment]
A few of the Effects aren't fully implements? I think this is better than nothing?

**Comment by dktapps** [comment]
Actually, it's worse than nothing, because it makes it harder to tell what's actually been fully implemented. This is the same reason why we don't accept partial implementations of blocks, items, mobs or anything else.

The partial-implemented things you see are things that were introduced prior to this policy being created, and are the reason why the policy was introduced.

**Comment by DenielWorld** [comment]
Slow falling basically exists as a negative levitation? Although, negative values are disallowed for effects in 4.0.0 I believe.

**Comment by dktapps** [comment]
Slow falling exists for the exact reason that vanilla doesn't allow negative effect levels.

**Pull request closed without merging**